src/applications/appeals/10182/tests/utils/focus.unit.spec.js

```diff
@@ -78,6 +78,7 @@ describe('focusIssue', () => {
   it.skip('should focus on second remove button', async () => {
     window.sessionStorage.setItem(LAST_ISSUE, '1,remove-cancel');
     const { container } = await renderPage();
+
```

Comment:
Just to match the duplicate test so it's easier to review for future refactoring.

Reply:
It's fine to remove this `focusIssue` test. Be bold young Padawan.